rollup.config.js

```diff
@@ -5,9 +5,13 @@ import resolve from '@rollup/plugin-node-resolve'
 import { terser } from 'rollup-plugin-terser'
 import pkg from './package.json'
 
+// Use peerDependencies as rollup external
+// https://rollupjs.org/guide/en/#peer-dependencies
 const { peerDependencies } = pkg
 const external = Object.keys(peerDependencies)
 
+// Explicitly specify unresolvable named exports.
+// https://github.com/rollup/plugins/tree/master/packages/commonjs#namedexports
```

Comment:
Why are these specifically causing problems that require being named exports? Wondering what you ran into to know this?

Reply:
For example, like `react-is`, it's main entry contains:

```js
'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react-is.production.min.js');
} else {
  module.exports = require('./cjs/react-is.development.js');
}
```

From the perspective of rollup, this module doesn't export `isForwardRef`(use in evergreen) explicitly, although it works, we should let rollup know how to resolve the exports. So we need to specify the namedExports.